API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/dashboard/domains.json

```
@@ -0,0 +1,85 @@
+              "blog_id": 106707880,
+              "subscription_id": null,
+              "can_manage_dns_records": false,
+              "can_manage_name_servers": false,
+              "can_update_contact_info": false,
+              "cannot_manage_dns_records_reason": null,
+              "cannot_manage_name_servers_reason": null,
+              "cannot_update_contact_info_reason": null,
+              "connection_mode": null,
+              "current_user_can_add_email": false,
+              "current_user_can_create_site_from_domain_only": true,
+              "current_user_cannot_add_email_reason": null,
+              "current_user_can_manage": true,
+              "current_user_is_owner": null,
+              "can_set_as_primary": true,
+              "domain": "testwebsite.wordpress.com",
+              "domain_notice_states": null,
+              "supports_domain_connect": null,
+              "email_forwards_count": 0,
+              "expiry": false,
+              "expiry_soon": false,
+              "expired": false,
+              "auto_renewing": false,
+              "pending_registration": false,
+              "pending_registration_time": "",
+              "has_registration": false,
+              "has_email_forward_dns_records": null,
+              "points_to_wpcom": true,
+              "privacy_available": false,
+              "private_domain": false,
+              "partner_domain": false,
+              "wpcom_domain": true,
+              "has_zone": false,
+              "is_renewable": false,
+              "is_redeemable": false,
+              "is_subdomain": true,
+              "is_eligible_for_inbound_transfer": false,
+              "is_locked": false,
+              "is_wpcom_staging_domain": false,
+              "transfer_away_eligible_at": null,
+              "type": "wpcom",
+              "registration_date": "",
+              "auto_renewal_date": "",
+              "google_apps_subscription": null,
+              "titan_mail_subscription": null,
+              "pending_whois_update": false,
+              "tld_maintenance_end_time": null,
+              "ssl_status": null,
+              "supports_gdpr_consent_management": false,
+              "supports_transfer_approval": false,
+              "domain_registration_agreement_url": "",
+              "contact_info_disclosure_available": false,
+              "contact_info_disclosed": false,
+              "renewable_until": null,
+              "redeemable_until": null,
+              "bundled_plan_subscription_id": null,
+              "product_slug": null,
+              "owner": "",
+              "pending_renewal": false,
+              "aftermarket_auction": false,
+              "aftermarket_auction_start": null,
+              "aftermarket_auction_end": null,
+              "nominet_pending_contact_verification_request": false,
+              "nominet_domain_suspended": false
+            }
+          ]
+      }
+    }
+  }
```

super nit: looks like it's missing the end of file line

---

Thanks, @jostnes, I'll add it now by suggesting myself a change 😆 

```suggestion
  }

```